locales/base/translation.json

```
@@ -77,6 +77,10 @@
       "phraseLocation": "An ideal place might be where you store other important documents & materials for safekeeping, that is not easily stolen and easy to retrieve. It's up to you!"
     }
   },
+  "walletSecurityPrimer": {
+    "title": "Keep your wallet safe",
+    "description": "Only you can access your wallet. If your phone is lost or stolen, Valora cannot recover it for you\n\nYou’ll need to set up a method to keep your wallet safe and protect your funds."
```

```suggestion
    "description": "Only you can access your wallet. If your phone is lost or stolen, Valora cannot recover it for you.\n\nYou’ll need to set up a method to keep your wallet safe and protect your funds."
```
looks like there's a period here in the figma designs